src/Extension.php

```diff
@@ -205,11 +214,19 @@ public function getConfigKey()
      */
     public function process(ContainerBuilder $container)
     {
-        $passes = $this->getCompilerPasses();
```

Comment:
This moves the main logic to a Controller, which is not good by design. Should the extension be called by other means than CLI (i.e. from another PHP library), it would break the functionality. The Cli Controllers should be used to extend the commands (or introduce new ones), rather than have all the Extension's logic inside.